keyboards/kyria/keymaps/plattfot/keymap.c

```diff
@@ -18,19 +18,26 @@
 
 enum custom_keycodes {
     PLACEHOLDER = SAFE_RANGE,  // can always be here
-    // Macros, where | is the cursor
-    M_LRBRC,  // [|]
-    M_LRCBR,  // {|}
-    M_LRPRN,  // (|)
-    M_LRABR,  // <|>
-    M_DQUOT,  // '|'
     // New keys
-    DBL_TAP,  // Repeat next key
+    CLO_TAP,  // Close the next key press
 };
 
-// Double Tap feature based on post from drashna
+// Close tap feature based on a post from drashna
 // https://www.reddit.com/r/olkb/comments/citkbx/double_key_press_modifier_qmkwould_work_like/ev9cue8/
-static bool double_tap_it = false;
+// Will insert the closing equivalent key and move the cursor inside.
+// For example clo_tap and then ( will result in (|), where | is the
+// cursor. For " it will be "|" as the close equivalent key is the
+// same key.
+enum close_tap_modes {
+  CLO_DISABLED = 0x000,
+  CLO_PRESSED  = 0x001, // Close tap key is pressed
+  CLO_ACTIVE   = 0x010, // Close tap next keypress
+  CLO_USED     = 0x100, // Turn off when close tap key is released
```

Comment:
Why not:
```suggestion
  CLO_DISABLED = 0b000,
  CLO_PRESSED  = 0b001, // Close tap key is pressed
  CLO_ACTIVE   = 0b010, // Close tap next keypress
  CLO_USED     = 0b100, // Turn off when close tap key is released
```
?

Reply:
I didn't know C supported `0b` prefix. Thanks for pointing that out. Update to using that in commit 2538722.